avocado/utils/software_manager.py

```diff
@@ -517,6 +537,95 @@ def provides(self, name):
         else:
             return None
 
+    def info(self, name):
+        """
+        Returns a name for the given package [name]
+
+        :param name: name of the package
+
+        :return full_name: name of the package with version & release
+        """
+        cmd = "%s info %s" % (self.base_command, name)
+        full_name = ""
+
+        try:
+            result = process.system_output(cmd)
+            for line in result.splitlines():
+                if line.startswith('Name'):
+                    full_name += "%s-" % line.split()[-1]
+                if line.startswith('Version'):
+                    full_name += "%s-" % line.split()[-1]
+                if line.startswith('Release'):
+                    full_name += "%s-" % line.split()[-1]
+                    return full_name
+        except process.CmdError, details:
+            log.error(details)
+            return ""
+
+    def build_dep(self, name):
+        """
+        Install build-dependencies for package [name]
+
+        :param name: name of the package
+
+        :return True: If build dependencies are installed properly
+        """
+        option = '-y --tolerant'
+        cmd = 'yum-builddep %s %s' % (option, name)
```

Comment:
could you please add `dnf` target for this? It uses `dnf builddep` and `--tolerant` is not available.

Reply:
@ldoktor Okay.  As the older redhat version does not provide `dnf`, this would be useful there. Can I add an additional `dnf builddep`?

Reply:
@ldoktor So you meant to add to dnf as well?

Reply:
If possible, yes. But looking at the code again it should work even now as long as `yum` is also installed (dnf backend is inherited from yum backend) so take this as a "bonus points" suggestion but not requirement.